user_tools/src/spark_rapids_pytools/resources/databricks_aws-configs.json

```diff
@@ -314,6 +314,71 @@
             "num_gpus": 4,
             "gpu_mem": 16384
           }
+        },
+        "g5.xlarge": {
```

Comment:
Should the g5* entries be inserted before the g4dn* entries to be treated as default?

Reply:
g5 entries need to be after g4dn entries because the current algorithm is like a best-fit search, and the later better-fit entry will override the previous ones.

Reply:
Oh right! I almost forgot what I wrote a year ago :)